MetaGenerator/ExceptionFormatter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Drawing;
+using Messerli.CommandLineAbstractions;
+using Messerli.MetaGeneratorAbstractions;
+using Pastel;
+
+namespace Messerli.MetaGenerator
+{
+    internal class ExceptionFormatter : IExceptionFormatter
+    {
+        private readonly IConsoleWriter _consoleWriter;
+        private readonly IGlobalOptions _globalOptions;
+        private readonly IExecutingPluginAssemblyProvider _executingPluginAssemblyProvider;
+
+        public ExceptionFormatter(IConsoleWriter consoleWriter, IGlobalOptions globalOptions, IExecutingPluginAssemblyProvider executingPluginAssemblyProvider)
+        {
+            _consoleWriter = consoleWriter;
+            _globalOptions = globalOptions;
+            _executingPluginAssemblyProvider = executingPluginAssemblyProvider;
+        }
+
+        public void FormatException(Exception exception)
+        {
+            _consoleWriter.WriteLine($"Exception in plugin: {Plugin()}");
```

Comment:
When there is no plugin, this reads as "Exception in plugin: MetaGenerator.exe". That's odd.

Reply:
Changed to module according to our discussion